.github/workflows/centos7-ci.yml

```diff
@@ -101,6 +101,7 @@ jobs:
     - name: Start CI env (PLUGIN_TEST)
       if: steps.test_env.outputs.type == 'plugin'
       run: |
+        sh t/plugin/openfunction/build-function-image.sh
```

Comment:
I think the image can be built by moving the files in `t/plugin/openfunction/function-example` to `ci/pod/openfunction`, and then building the image, like: https://github.com/apache /apisix/blob/master/ci/pod/docker-compose.first.yml#L94. This makes it easier for us to manage all test images later. cc @spacewander

Reply:
Sure, it is good to move files in t/plugin/openfunction/function-example to ci/pod/openfunction, but building images of function in OpenFunction requires  using build-packs not dockerfile .